EngineLayer/Psm.cs

```diff
@@ -401,14 +401,14 @@ internal void Add(Psm psmParent)
         private Tuple<string, double?> Resolve(IEnumerable<double> enumerable)
         {
             double doubleTol = 1e-6;
-            var list = enumerable.ToList();
+            var list = enumerable.Distinct().ToList();
```

Comment:
I don't think this should be Distinct(). 
The whole idea is to either collapse all into one, or display all the options! Say you have 3 ambiguous peptides with proteins 1,1,2 
In this case just saying 1,2 loses the assignment information